Add custom instrumentation link to .NET datastore instrumentation info (#19084)
newrelic/docs-website: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements.mdx

            The .NET agent does not directly monitor datastore processes. Also, the .NET SQL parameter capture in a query trace does not list parameters for a parameterized query or a stored procedure.

            Collecting [instance details](/docs/apm/applications-menu/features/analyze-database-instance-level-performance-issues) for supported datastores is enabled by default. To request instance-level information from datastores not currently listed, get support at [support.newrelic.com](https://support.newrelic.com).

>             If your datastore is not listed here, you can add custom instrumentation using the `RecordDatastoreSegment` method in the [.NET agent API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#ITransaction).

Reviewer: ```suggestion
            If your datastore isn't listed here, you can add custom instrumentation using the `RecordDatastoreSegment` method in the [.NET agent API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#ITransaction).
```

                </tbody>
                </table>

>                 If your datastore is not listed here, you can add custom instrumentation using the `RecordDatastoreSegment` method in the [.NET agent API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#ITransaction).

Reviewer: ```suggestion
                If your datastore isn't listed here, you can add custom instrumentation using the `RecordDatastoreSegment` method in the [.NET agent API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#ITransaction).
```
